src/services/notifier/hooks/plans.hook.ts

```diff
@@ -61,19 +54,15 @@ export default {
         } = query
 
         if (channels?.length) {
-          paramsSeq.include = [
-            ...paramsSeq.include,
-            {
-              model: NotifierChannelModel,
-              as: 'channels',
-              required: true,
-              attributes: ['name'],
-              where: {
-                name: channels
+          paramsSeq.where = {
+            ...paramsSeq.where ?? {},
+            channels: {
+              [Op.contains]: {
+                type: channels
```

Comment:
Considering we are storing origin also as part of json, wouldn't this query return non-relevant results for ex. if email address is apiuser@gmail.com or if the url is https://email.domain.com

Reply:
no this is just to filter channel types like API, EMAIL, etc

Reply:
but it's also not used so I might just remove it.